src/sizes.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * include/linux/sizes.h
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
```

Comment:
Are we GPL now? :)

Reply:
Nope, completely rewrote it from scratch from memory this time around :D